app/assets/javascripts/modules/track-responses.js

```diff
@@ -9,8 +9,8 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
       track(element[0])
     }
 
-    function getQuestionKey (submittedForm) {
-      return submittedForm.getAttribute('data-question-key')
+    function getQuestionHeading (submittedForm) {
+      return submittedForm.querySelector('h1').innerText
```

Comment:
Not sure I agree with the approach here. The advantage of using a data attribute is that it communicates the element is being used to pick data from, regardless of its semantics (now is a heading level 1, but tomorrow could be a heading level 2 – and the analytics script will fail silently).

Was expecting something like `querySelector('data-question-key').innerText`

Reply:
Thats a good point. I'll add in another attribute that's data-question-text to support the existing events, without referencing the h1 e.g. `querySelector('data-question-text').innerText`